frontend/src/Core/service/PublicUsersService.js

```diff
@@ -0,0 +1,9 @@
+export default class PublicUsersService{
+    constructor(httpRequest) {
+        this.httpRequest = httpRequest;
+    }
+
+    getUser(id){
+        return this.httpRequest.get(`users/${id}`).then(data => data);
```

Comment:
Ten end point ```users/``` dobrze by było zapisać do jakiejś zmiennej.
Teraz jest to używane tylko raz, ale w przyszłości pewnie pojawi się też jakaś metoda POST i też będzie trzeba użyć tego endpointu i głupio tak za każdym razem wpisywać to magicznie.
A jakby tego było więcej i by się nagle zmieniła ścieżka, to dopiero byłaby zabawa z zmienianie tego wszędzie.
A tak to tylko jedna zmiana w zmiennej i zrobione :)